packages/composites/anchor/package.json

```diff
@@ -34,6 +34,7 @@
     "@chainlink/ea-test-helpers": "workspace:*",
     "@chainlink/terra-view-function-adapter": "workspace:*",
     "@chainlink/token-allocation-adapter": "workspace:*",
+    "bignumber.js": "^9.0.0",
     "ethers": "^5.5.1",
     "tslib": "^2.3.1"
```

Comment:
Mind if we tie this to these specific versions? I would like to start avoiding `^` for package versions if possible